### ISSUE TITLE
Carthage installation failed

### ISSUE THREAD

**Issue by parvezunlimint** [piwik/piwik-sdk-ios#440]
I am trying to add Matomo ios SDK to my project. I am getting error 

When I call 
carthage update 

Error: 
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/parvez/Desktop/Carthage/Demo/Carthage/Checkouts/matomo-sdk-ios/MatomoTracker.xcworkspace -scheme MatomoTracker -configuration Release -derivedDataPath /Users/parvez/Library/Caches/org.carthage.CarthageKit/DerivedData/15.0_15A240d/matomo-sdk-ios/v7.5.2 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive VALIDATE_WORKSPACE=NO -archivePath /var/folders/gp/2413kxj924z22fcdx4p64vrh0000gn/T/matomo-sdk-ios SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/parvez/Desktop/Carthage/Demo/Carthage/Checkouts/matomo-sdk-ios)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/gp/2413kxj924z22fcdx4p64vrh0000gn/T/carthage-xcodebuild.JM7dCt.log

When I check the log 

/Users/parvez/Desktop/Carthage/Demo/Carthage/Checkouts/matomo-sdk-ios/MatomoTracker.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 12.0 to 17.0.99. (in target 'MatomoTracker' from project 'MatomoTracker')
** ARCHIVE FAILED **

**Comment by brototyp** [comment]
Hi @parvezunlimint, thanks for creating this ticket. Last time I looked into the Carthage support for Matomo I stumbled over [this issue](https://github.com/Carthage/Carthage/issues/3146) which I struggled with to resolve. This led me to comment out the automated Carthage Build tests so I don't know too well about the current Carthage support tbh.

But, looking at your error there, I noticed one thing: Is it possible that the project you are trying to integrate Matomo into supports iOS 8.0 and above? That might be an issue, since Matomo doesn't support iOS anymore.
